packages/vega-spec-builder/src/types/specUtil.types.ts

```diff
@@ -34,11 +34,14 @@ export interface ScSpec extends Spec {
 	 * Parsing the spec for information about the bar is brittle, since implementation details may change.
 	 * A preferred approach would be to add the needed information to usermeta { orientation: 'vertical' }.
 	 */
-	usermeta?: {
-		orientation?: Orientation;
-	};
+	usermeta: UserMeta;
 }
 
+export type UserMeta = {
+	orientation?: Orientation;
```

Comment:
Can we rename this to be a bit more specific? Even something like `chartOrientation`. We could easily support orientation on things like axis labels, legends etc.